tgui/packages/tgui/interfaces/AmmoWorkbench.jsx

```diff
@@ -153,9 +153,9 @@ export const AmmunitionsTab = (props) => {
       </Section>
       {!!hacked && (
         <NoticeBox textAlign="center" color="bad">
-          !WARNING! - ARMADYNE SAFETY PROTOCOLS ARE NOT ENGAGED! MISUSE IS NOT
+          !WARNING! - WORKBENCH SAFETY PROTOCOLS ARE NOT ENGAGED! MISUSE IS NOT
           COVERED UNDER WARRANTY. SOME MUNITION TYPES MAY CONSTITUTE A WAR CRIME
-          IN YOUR AREA. PLEASE CONTACT AN ARMADYNE ADMINISTRATOR IMMEDIATELY.
+          IN YOUR AREA. PLEASE CONTACT A NANOTRASEN ADMINISTRATOR IMMEDIATELY.
```

Comment:
Just a nagging worry this might be taken a bit too meta and thought to be a UI bug... Usually see 'NT Admin' in reference to CC, in turn literal admins, yada yada...
Might just be a worse-case-scenario mindset though.
```suggestion
          IN YOUR AREA. PLEASE CONTACT A CUSTOMER SERVICE REPRESENTATIVE IMMEDIATELY.
```